README.md

```diff
@@ -860,6 +861,16 @@ ANSWERS.addComponent('VerticalResults', {
 })
 ```
 
+## Results Count Component
```

Comment:
Since the Results Count component only works with the vertical results server response, do you think it would make sense to have "vertical" in its name?

Reply:
ah yeah I like that, updated!